admin/class-admin-asset-manager.php

```diff
@@ -452,21 +513,67 @@ protected function scripts_to_be_registered() {
 				'src'  => 'analysis-' . $flat_version,
 				'deps' => [
 					'lodash',
+					'wp-autop',
+					self::PREFIX . 'feature-flag',
+					self::PREFIX . 'jed',
 					self::PREFIX . 'commons',
 				],
 			],
 			[
+				/**
+				 * Asset for backwards-compatibility, to make sure
+				 * the addons don't break when we change dependencies.
+				 */
 				'name' => 'components',
-				'src'  => 'components-' . $flat_version,
+				'src'  => false,
 				'deps' => [
-					self::PREFIX . 'jed',
-					self::PREFIX . 'redux',
-					self::PREFIX . 'analysis',
+					self::PREFIX . 'feature-flag',
+					self::PREFIX . 'helpers',
+					self::PREFIX . 'style-guide',
+					self::PREFIX . 'configuration-wizard-package',
+					self::PREFIX . 'analysis-report',
+					self::PREFIX . 'components-modern',
+					self::PREFIX . 'replacement-variable-editor',
+					self::PREFIX . 'search-metadata-previews',
+					self::PREFIX . 'social-metadata-forms',
+					self::PREFIX . 'components-legacy',
+				],
+			],
+			[
+				// The `@yoast/components` package.
+				'name' => 'components-modern',
```

Comment:
If we have `components-legacy`, this can just be `components`

Reply:
We currently have a `components` asset; which contains all the packages.
I didn't think it would be wise to just replace the contents of the asset with a small subset of what it currently is.

Following your suggestion to rename the legacy components, I suggest we use `yoast-components` for this asset.